Coral Setpoints

MicahCoder opened the pull request:
Creates setpoints for the Coral Subsystem, with a relative for elevator and a absolute for wrist. 
![image](https://github.com/user-attachments/assets/7c929774-ad4a-4905-82d8-f86c31aa26dc)

rakdosintolerant committed:
INIT

rakdosintolerant committed:
INIT

rakdosintolerant committed:
Limit Switch

rakdosintolerant committed:
Limit Switch

rakdosintolerant committed:
relative encoders

rakdosintolerant committed:
relative encoders

MicahCoder committed:
Angle, Integrated CMDS

MicahCoder committed:
Angle, Integrated CMDS

MicahCoder committed:
Homing

MicahCoder committed:
Homing

MicahCoder committed:
AngleCommands

MicahCoder committed:
AngleCommands

MicahCoder committed:
Arm Setpoints

MicahCoder committed:
Arm Setpoints

MicahCoder committed:
Merge branch 'main' into LimitSwitchRelativeCoral

E-Latsis reviewed (approved):
Looks great. We'll test this today and then talk about possibly adding more set points for the wrist in class

E-Latsis committed:
Unused PID values removed

ipex200 committed:
i organized all the constants (#38)

* i organized all the constants

i put all the constants required by a certain subsystem in a file for each subsystem so elevator has its own constants ,drive has its own , coralintake has its own ect...

* i fixed something

* CoralIntake and elevator are now their own folder

For organization purposes: Coral Subsystem folder is gone and the elevator and coral Intake files are in the subsystems folder

---------

Co-authored-by: E-Latsis <147000359+E-Latsis@users.noreply.github.com>

MicahCoder committed:
Commit

MicahCoder committed:
Commit

MicahCoder committed:
Logging Setpoint

MicahCoder committed:
Logging Setpoint

E-Latsis committed:
Right Encoder

E-Latsis committed:
Rebased this branch so I can finally merge it
